static/css/section/_desktop.scss

```diff
@@ -811,12 +593,12 @@ $item_margin: 40px;
     }
 
     @media only screen and (min-width: $breakpoint-large) {
```

Comment:
Could you move these two `@media` statements so they're both inside `.four-col::after`?